roles/openshift_node_group/files/sync.yaml

```diff
@@ -119,6 +119,10 @@ spec:
             fi
 
             # detect whether the node-config.yaml has changed, and if so trigger a restart of the kubelet.
+            if [[ ! -f /etc/origin/node/node-config.yaml ]]; then
+              cat /dev/null > /tmp/.old
+            fi
```

Comment:
Let's an else here and always has the file if it exists just to cover all cases.